examples/SampleApp/android/settings.gradle

```diff
@@ -1,8 +1,6 @@
 rootProject.name = 'SampleApp'
 apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
 include ':app'
-include ':react-native-video'
```

Comment:
This step is not needed anymore?

Reply:
No, with 6.0.0-alpha-2 everything is by default exoplayer, so you don't need this anymore.